### PR TITLE
feat(t8s-cluster/clusterClass)!: configure static machineDeploymentClasses instead of one per computePlane

### LINKED ISSUE #584: [t8s-cluster] cannot delete compute plane
When deleting a compute-plane, the first time fails as the ClusterClass still requires the class to exist as the CAPI cluster still needs it before the t8s-engine-operator updates the CAPI cluster

The second reconciliation works tho.

So we kinda have to have static classes that are more CAPI-templated 🤮

### PR DESCRIPTION
**Pull request by cwrau**
Closes #584

Needs [gitlab.teuto.net/4teuto/ops/k8s/t8s-engine/-/merge_requests/661](https://gitlab.teuto.net/4teuto/ops/k8s/t8s-engine/-/merge_requests/661)

### CODE REVIEW

File: charts/t8s-cluster/ci/gpu-flavor-values.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-workers:
+computePlanes:
```
Comment (marvinWolff) [inline charts/t8s-cluster/ci/gpu-flavor-values.yaml:1]: Why is this renamed?

Comment (cwrau) [reply charts/t8s-cluster/ci/gpu-flavor-values.yaml]: Because of https://gitlab.teuto.net/4teuto/ops/k8s/t8s-engine/-/merge_requests/663

File: charts/t8s-cluster/templates/management-cluster/clusterClass/clusterClass.yaml [hunk 2]
```diff
@@ -16,123 +22,125 @@ spec:
         - status: 'False'
           timeout: 600s
           type: Ready
-    nodeDrainTimeout: {{ .Values.controlPlane.nodeDrainTimeout | default "8m" }}
```
Comment (marvinWolff) [inline charts/t8s-cluster/templates/management-cluster/clusterClass/clusterClass.yaml:19]: Why remove this?

Comment (marvinWolff) [reply charts/t8s-cluster/templates/management-cluster/clusterClass/clusterClass.yaml:19]: Ok, cleared

**Review by tasches — APPROVED** [review]
tested and reviewed. LGTM
